### ISSUE TITLE
[Request]: Any plans for a React stack?

### ISSUE THREAD

**Issue by perryraskin** [uicrooks/shopify-theme-lab#127]
### Operating system

macOS

### Node.js version

14.0.0

### Shopify CLI version

2.0.0

### Version

4 (Current)

### Details

Would absolutely love a React version of this. Happy to discuss especially if you want help with it!

### Notice

- [X] I read the notice

**Comment by sergejcodes** [comment]
Not really, but I could add a detailed React section to the README/Docs, it shouldn't take too long to adjust everything for React.

There's a discussion regarding React [here](https://github.com/uicrooks/shopify-theme-lab/discussions/74)
One thing that's missing is a proper way to handle props and slots. I think there's a need for some custom code to make them work.

On another note: what are some compelling reasons to use React for a Shopify online store? It's not a SPA, so are there any benefits over a framework like Alpine.js which doesn't require any js components at all?

**Comment by perryraskin** [comment]
My bad, forgot to check the Discussions tab! For me it's simply bc React is what I'm most familiar with over the others. I do believe many others are in a similar position, too.

Thanks!

**Comment by smartbadger** [comment]
@sergejcodes I've never heard of Alpine, I'll check it out.

I'm curious to know your opinion. Would Alpine be better suited to use here instead of Vue? Would you rather use Alpine than Vue? I've built more in React than Vue and nothing in Alpine. However, if it's better to use Vue or Alpine, I'd rather pick the right tool for the job.

**Comment by Kusou1** [comment]
react can see there 
[https://shopify.dev/api/hydrogen](url)
I feel that support should be better, but currently only a preview

**Comment by youngbloodk** [comment]
This.
